anypoint-signin-aware.js

```diff
@@ -296,7 +304,7 @@ export const AnypointAuth = {
       AnypointAuth.setAuthData();
       return;
     }
-    return AnypointAuth._getProfile(info.accessToken).catch(() => AnypointAuth.setAuthData(info.accessToken));
+    return Promise.resolve(AnypointAuth.setAuthData(info.accessToken));
```

Comment:
This function seems to either return a promise or an undefined value. This is a dangerous practice.

Reply:
It should be marked as `async` and it won't matter

Reply:
It doesn't need to return anything anymore since we are no longer fetching the profile